From e70f68a55644d798d70cb1c902b640093c40f02b Mon Sep 17 00:00:00 2001
From: Philippe Normand <philn@igalia.com>
Date: Wed, 23 Apr 2025 14:17:37 +0100
Subject: [PATCH] Revert "video-overlay-composition: Declare the
 video/size/orientation tags for the meta"

This reverts commit d0c158fb23c3cb6f8b551d55a3d0a72e9dc24b6b.
---
 .../gst-libs/gst/video/video-overlay-composition.c           | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/subprojects/gst-plugins-base/gst-libs/gst/video/video-overlay-composition.c b/subprojects/gst-plugins-base/gst-libs/gst/video/video-overlay-composition.c
index e0309ae3e2..e7e1f55028 100644
--- a/subprojects/gst-plugins-base/gst-libs/gst/video/video-overlay-composition.c
+++ b/subprojects/gst-plugins-base/gst-libs/gst/video/video-overlay-composition.c
@@ -284,10 +284,7 @@ GType
 gst_video_overlay_composition_meta_api_get_type (void)
 {
   static GType type = 0;
-  static const gchar *tags[] =
-      { GST_META_TAG_VIDEO_STR, GST_META_TAG_VIDEO_ORIENTATION_STR,
-    GST_META_TAG_VIDEO_SIZE_STR, NULL
-  };
+  static const gchar *tags[] = { NULL };
 
   if (g_once_init_enter (&type)) {
     GType _type =
-- 
2.49.0

